apps/gauzy/src/app/pages/settings/oauth-clients/oauth-client-secret-dialog/oauth-client-secret-dialog.component.html

```
@@ -0,0 +1,42 @@
+				<code>{{ client.clientId }}</code>
+				<button nbButton size="small" status="primary" ghost (click)="copyClientId()">
+					<nb-icon icon="copy-outline"></nb-icon>
+				</button>
+			</div>
+		</div>
+
+		<div class="credential">
+			<label>{{ 'OAUTH_CLIENTS.SECRET_DIALOG.CLIENT_SECRET' | translate }}</label>
+			<div class="credential-row">
+				<code class="secret">{{ client.clientSecret }}</code>
+				<button nbButton size="small" status="primary" ghost (click)="copySecret()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)